jac-uk/apply
# Feature/1242 2FA - Replace Phone Number With UK Mobile Phone Number

## What's included?

Closes #1242 

- Add `MobileNumber` and `MobileVerificationModal` components.
- Add the UK mobile phone number validation using `libphonenumber-js`.


## Who should test?
✅ Product owner
✅ Developers

## How to test?

[Preview URL](https://jac-apply-develop--pr1245-feature-1242-2fa-rep-fhokfih3.web.app/)

1. Log in via the preview URL and go to the "Your profile" page.
2. Edit the "UK Mobile phone number" field and check if the validation is working.
3. Apply for an application and go to the "Personal details" section.
4. Check if there is a link next to the "UK Mobile phone number" field which will redirect to the profile page. 

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

https://github.com/user-attachments/assets/64be3d3c-ca05-486e-b375-f8e46b83d368

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
